src/utils/utils.py

```
@@ -239,3 +227,82 @@ async def capture_screenshot(browser_context):
+        active_page = pages[0]
+        for page in pages:
+            if page.url != "about:blank":
+                active_page = page
+    else:
+        return None
+
+    # Get the DOM of the page
+    try:
+        dom = await active_page.content()
+        return dom
+    except Exception as e:
```

Ruff (0.8.2)

Do not catch blind exception: `Exception`

(BLE001)